soaks/bin/analyze_experiment

```diff
@@ -83,7 +78,13 @@ for exp in csv.experiment.unique():
     # samples actually have the same mean -- are from the same distribution --
     # and so there's some statistically interesting difference between the two
     # samples. For our purposes here that implies that performance has changed.
-    res = scipy.stats.ttest_ind(baseline['value'], comparison['value'], equal_var=False, trim=trim)
```

Comment:
could we just set `nan_policy` = `omit` here? I guess no real difference though, just an alternative workaround. 
Though wondering where NaN in the data itself come from?

Reply:
That's what's weird here. The underlying data has no NaN values -- I made sure of that by inspection -- so it's either an undocumented edge condition in scipy or a bug on their part. I don't read python well enough to understand the small handful of bugs that might describe this, but I'll dig in some more and file upstream.